gtsam/geometry/Unit3.cpp

```diff
@@ -80,12 +80,10 @@ static Point3 CalculateBestAxis(const Point3& n) {
 
 /* ************************************************************************* */
 const Matrix32& Unit3::basis(OptionalJacobian<6, 2> H) const {
-#ifdef GTSAM_USE_TBB
```

Comment:
I would consider leaving these `#ifdef GTSAM_USE_TBB` guards in place. They have the benefit of making single-threaded builds (without TBB) completely mutex-free.

Reply:
hmm, I see, yeah that would be useful. But I don't like keeping the TBB name since it is not a TBB function/class. Maybe renaming the guard to `GTSAM_MULTITHREADED` would be more accurate?

Reply:
I agree it would be more accurate, but let's keep that change until TBB is not the only way to do multi-threading. A change like this has the potential to break some setups of users in ways we can't anticipate. Better to do this as part of a version bump with release docs - I propose this (awesome!) PR should simply addresses the warnings.

Reply:
Okay I'll add the guards back, but I really hope we don't forget. Maybe we can keep an issue and PR open specifically for this until we decide to add it in.